tests/e2e/data/help-contents/ligo-contents.ts

```diff
@@ -15,8 +15,8 @@ Commands:
                                    from the abstractions provided by Taqueria
   taq compile <sourceFile>        Compile a smart contract written in a LIGO syn
```

Comment:
This output is getting a bit long, can we trim this as much as possible?

Reply:
done